wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - nominations
+  - nom
+  - noms
+  - heart
+---
+
+# Nomination
+
+*See also: [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)*
+
+::: Infobox
+![](img/nomination-progress-beatmapset1857663.png "Beatmap ranking status displayed on the discussion page, with hype and nomination progress bars partially filled.")
+:::
+
+**Nomination** is a formal vote of approval coming from [beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) or members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT), which indicates a beatmap's readiness to become [ranked](/wiki/Beatmap/Category#ranked). It is given to completed beatmaps of sufficient quality that follow the [ranking criteria](/wiki/Ranking_Criteria) and have 5 or more [hypes](/wiki/Beatmap/Hype).
+
+Beatmaps that receive two nominations per each available [game mode](/wiki/Game_mode) are moved to the [beatmap ranking queue](/wiki/Beatmap_ranking_procedure/Ranking_queue), where they stay for at least 7 days for further inspection. If no critical issues are found, beatmaps are then moved to the Ranked category and receive their own [leaderboards](/wiki/Ranking).
+
+Nominations may be removed from a map in one of the following cases:
+
+- A beatmap is updated by its [host](/wiki/Beatmap/Beatmap_host).
+- Serious or unrankable issues found by a BN or a member of the NAT.
+
+## History
+
+Nominations in their present form were implemented as part of the [beatmap discussion](/wiki/Beatmap_discussion) system.
+
+Previously, in the era of [forum modding](/wiki/Modding/Forum_modding), nominations were expressed through a combination of ![Bubble icon](/wiki/shared/icon/bubble.gif) [bubble](/wiki/Modding/Bubble) and ![Heart icon](/wiki/shared/icon/heart.gif) rank posts, where bubble icon meant the first nomination, and the heart icon was the second. Maps going for [approval](/wiki/Beatmap/Category#approved) instead of rank needed to abide by a slightly different set of rules and required three nominations, with the last of them having ![Flame icon](/wiki/shared/icon/flame.gif) flame icon.
```

Comment:
```suggestion
Previously, in the era of [forum modding](/wiki/Modding/Forum_modding), nominations were expressed through a combination of ![Bubble icon](/wiki/shared/icon/bubble.gif) [bubble](/wiki/Modding/Bubble) and ![Heart icon](/wiki/shared/icon/heart.gif) rank posts, where a bubble icon meant the first nomination, and the heart icon was the second. Maps going for [approval](/wiki/Beatmap/Category#approved) instead of ranking needed to abide by a slightly different set of rules and required three nominations, with the last of them having a ![Flame icon](/wiki/shared/icon/flame.gif) flame icon.
```